src/niswitch/system_tests/test_system_niswitch.py

```diff
@@ -141,3 +144,20 @@ def test_functions_get_error_description(session):
     description = session.get_error_description(0)   # expect no errors
     assert description == ''
 
+
+def test_functions_connect_disconnect_multiple(session):
+    session.connect_multiple('c0->r0, c0->r1')   # expect no errors
+    session.disconnect_multiple('c0->r0, c0->r1')   # expect no errors
+
+
+def test_functions_disable(session):
+    channel1 = 'c0'
+    channel2 = 'r0'
+    session.connect(channel1, channel2)
+    session.disable()   # expect no errors
+    assert session.can_connect(channel1, channel2) == niswitch.PathCapability.PATH_AVAILABLE
+
+
+def test_functions_interchange(session):
+    session.clear_interchange_warnings()   # expect no errors
```

Comment:
This applies to all our APIs. They are sort of useless. How about we stop generating?

Reply:
I agree that there is little use for them. But we do support them in both LabVIEW and LabVIEW NXG for all drivers. Removing them here would be inconsistent with that. 
Thoughts?

Reply:
Opened https://github.com/ni/nimi-python/issues/342 so we can discuss there.